src/ComboboxInput/ComboboxInput.js

```diff
@@ -61,7 +50,8 @@ ComboboxInput.propTypes = {
     disableStyles: PropTypes.bool,
     inputProps: PropTypes.object,
     placeholder: PropTypes.string,
-    popoverProps: PropTypes.object
+    popoverProps: PropTypes.object,
+    state: PropTypes.oneOf(FORM_STATES)
 };
 
 ComboboxInput.propDescriptions = {
```

Comment:
it's a separate issue, but should all props have a corresponding propDescription?  Should you be adding one for `state`?

Reply:
There's a global prop definition for state since it's shared amongst so many components.